### ISSUE TITLE
8.0.8- Sensor Clipping Overlay

### ISSUE THREAD

**Issue by disfear87** [mirsadm/motioncam#190]
When the overlay is turned on, after one recording the overlay dissapears, when I turn off the overlay switch the overlay reappears (i.e: the switch reverse toggles till app restart) (Xiaomi 11 Lite NE 5G)

**Comment by mirsadm** [comment]
Yep at the moment I'm doing that to prevent any dropped frames during recording. I'll see if I can keep it on during recording
